jmnzdz/movies-app
# Feature/movies component

### Feature

Create a new component to manage the existing movies. This feature add the following functionality:

- List all the existing movies.
- Show the movie information (clicking in his poster).
- Create new movies.
- Edit the movie information.
- Remove a movie (confirmation needed).

### Testing

Movie list:
- See the movie title in each card.
- See the movie genre with the pattern '#genre'.
- See the default image for movies without posters.
